src/dbus_mpris.rs

```diff
@@ -698,6 +727,24 @@ async fn create_dbus_server(
     }
 }
 
+fn get_device_id(sp_client: &Arc<AuthCodeSpotify>, device_name: &String, only_active: bool) -> Option<Option<String>> {
+    let device_result = sp_client.device();
+    return match device_result {
```

Comment:
You can use `match` directly and omit the semicolon at the end, and it should work as well. ;)

Reply:
Fixed